rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-nodes-about-autoscaling-nodes"]
+= About autoscaling nodes on a cluster
+:context: rosa-nodes-about-autoscaling-nodes
+toc::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+Autoscaling is available only on clusters that were purchased through the Red Hat Marketplace.
+====
+endif::[]
+
+The autoscaler option can be configured to automatically scale the number of machines in a cluster.
+
+The cluster autoscaler increases the size of the cluster when there are pods that failed to schedule on any of the current nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+
+Additionally, the cluster autoscaler decreases the size of the cluster when some nodes are consistently not needed for a significant period, such as when it has low resource use and all of its important pods can fit on other nodes.
+
+When you enable autoscaling, you must also set a minimum and maximum number of worker nodes.
+
+[NOTE]
+====
+Only cluster owners and organization admins can scale or delete a cluster.
+====
+
+[id="nodes-enabling-autoscaling-nodes"]
+== Enabling autoscaling nodes on a cluster
+
+You can enable autoscaling on worker nodes to increase or decrease the number of nodes available by editing the machine pool definition for an existing cluster.
+
+[discrete]
+include::modules/ocm-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+
+ifdef::openshift-rosa[]
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../../rosa_getting_started/rosa_getting_started_iam/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
+====
+
+[discrete]
+include::modules/rosa-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+endif::[]
+
+[id="nodes-disabling-autoscaling-nodes"]
+== Disabling autoscaling nodes on a cluster
```

Comment:
If the world were perfect, this would be a separate assembly, and you could retitle the current assembly to better reflect its contents. (It sounds like a conceptual assembly, not a procedural one.)